File: Python/NumPy/npwhere/npwhere.py
import numpy as np
import pandas as pd

df = pd.read_csv(
    'https://raw.githubusercontent.com/krishnaik06/All-Hyperparamter-Optimization/master/diabetes.csv')
df['Glucose'] = np.where(
    df['Glucose'] == 0, df['Glucose'].median(), df['Glucose'])
df['Insulin'] = np.where(
    df['Insulin'] == 0, df['Insulin'].median(), df['Insulin'])

File: Python/NumPy/npwhere/npwhere.jl
using CSV, DataFrames, Statistics

df = CSV.read(download("https://raw.githubusercontent.com/krishnaik06/All-Hyperparamter-Optimization/master/diabetes.csv"), DataFrame)
#df[!, :Insulin] = Float64.(df[:, :Insulin])
df[!, :Insulin] = replace!(Float64.(df.Insulin), 0 => Statistics.median(df.Insulin))
df[!, :Glucose] = replace!(Float64.(df.Glucose), 0 => Statistics.median(df.Glucose))
println(df)